packages/rulesets/src/spectral/az-common.ts

```diff
@@ -30,6 +42,138 @@ const ruleset: any = {
+      message: "{{error}}",
+      severity: "error",
+      resolved: true,
+      formats: [oas2],
+      given: ["$[paths,'x-ms-paths'].*.*[?(@property === 'operationId')]"],
+      then: {
+        function: operationIdNounConflictingModelNames,
+      },
+    },
+    OperationIdNounVerb: {
+      description: "OperationId should be of the form `Noun_Verb`.",
+      message: "{{error}}",
+      severity: "error",
+      resolved: true,
+      formats: [oas2],
+      given: ["$[paths,'x-ms-paths'].*.*[?(@property === 'operationId')]"],
+      then: {
+        function: operationIdNounVerb,
+      },
+    },
+    OperationIdSingleUnderscore: {
+      description: "An operationId can have exactly one underscore, not adhering to it can cause errors in code generation.",
+      message: "{{error}}",
+      severity: "error",
+      resolved: true,
+      formats: [oas2],
+      given: ["$[paths,'x-ms-paths'].*.*[?(@property === 'operationId')]"],
+      then: {
+        function: operationIdSingleUnderscore,
+      },
+    },
+    GetInOperationName: {
+      description: "Verifies whether value for `operationId` is named as per ARM guidelines.",
+      message: "{{error}}",
+      severity: "error",
+      resolved: true,
+      formats: [oas2],
+      given: ["$[paths,'x-ms-paths'].*[get][?(@property === 'operationId')]"],
+      then: {
+        function: getInOperationName,
+      },
+    },
+    PutInOperationName: {
+      description: "Verifies whether value for `operationId` is named as per ARM guidelines.",
+      message: "{{error}}",
+      severity: "error",
+      resolved: true,
+      formats: [oas2],
+      given: ["$[paths,'x-ms-paths'].*[put][?(@property === 'operationId')]"],
+      then: {
+        function: putInOperationName,
+      },
+    },
+    PatchInOperationName: {
+      description: "Verifies whether value for `operationId` is named as per ARM guidelines.",
+      message: "{{error}}",
+      severity: "error",
+      resolved: true,
+      formats: [oas2],
+      given: ["$[paths,'x-ms-paths'].*[patch][?(@property === 'operationId')]"],
+      then: {
+        function: patchInOperationName,
+      },
+    },
+    DeleteInOperationName: {
+      description: "Verifies whether value for `operationId` is named as per ARM guidelines.",
+      message: "{{error}}",
+      severity: "error",
+      resolved: true,
+      formats: [oas2],
+      given: ["$[paths,'x-ms-paths'].*[delete][?(@property === 'operationId')]"],
+      then: {
+        function: deleteInOperationName,
+      },
+    },
+    ParameterNotDefinedInGlobalParameters: {
+      description: "Per ARM guidelines, if `subscriptionId` is used anywhere as a path parameter, it must always be defined as global parameter. `api-version` is almost always an input parameter in any ARM spec and must also be defined as a global parameter.",
+      message: "{{error}}",
+      severity: "error",
+      resolved: true,
```

Comment:
resolved:false

Reply:
But it will introduce a lot of false alarm for this rule, as in the resolved swagger, all the parameters are inlined